app/src/types.ts

```diff
@@ -101,8 +101,8 @@ export interface Custom {
 }
 
 interface Description {
-  custom: Record<string, string>;
-  display: string;
+  custom: Record<string, string> | null;
+  display: string | null;
```

Comment:
it doesn't look to me like Description is actually being displayed anywhere any more?

Reply:
yeah its not but will leave in as still part of data structure recieved